docs/getting-started.md

```diff
@@ -364,6 +364,23 @@ not run on an EC2 .metal instance. You can skip performance tests with:
  ./tools/devtool test -- --ignore integration_tests/performance
  ```
 
+## Troubleshooting
+
+#### Errors while using `curl` to access the API
+
+Points to check to confirm the API socket is running and accessible:
+* Check that the user running the Firecracker process and the user using `curl` have equivalent privileges. 
+  For example, if you run Firecracker with **sudo** that you run `curl` with **sudo** as well
```

Comment:
Shouldn't r/w access to the socket be sufficient? I.e. `[ -w SOCKET ] && echo "Good to go"`.